<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateForm</class>
 <widget class="QWidget" name="TemplateForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="GridTemplateButton" name="templateButton">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="descriptionEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="editButton">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="eraseButton">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GridTemplateButton</class>
   <extends>QPushButton</extends>
   <header location="global">UI/gridtemplatebutton.h</header>
   <slots>
    <slot>apply()</slot>
    <slot>edit()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>templateButton</tabstop>
  <tabstop>descriptionEdit</tabstop>
  <tabstop>editButton</tabstop>
  <tabstop>applyButton</tabstop>
  <tabstop>eraseButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>editButton</sender>
   <signal>clicked()</signal>
   <receiver>descriptionLabel</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editButton</sender>
   <signal>clicked()</signal>
   <receiver>descriptionEdit</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editButton</sender>
   <signal>clicked()</signal>
   <receiver>editButton</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>applyButton</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>descriptionEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>descriptionLabel</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>descriptionLabel</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>eraseButton</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>editButton</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>descriptionEdit</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editButton</sender>
   <signal>clicked()</signal>
   <receiver>applyButton</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editButton</sender>
   <signal>clicked()</signal>
   <receiver>eraseButton</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editButton</sender>
   <signal>clicked()</signal>
   <receiver>templateButton</receiver>
   <slot>edit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>templateButton</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eraseButton</sender>
   <signal>clicked()</signal>
   <receiver>TemplateForm</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
